### DISCUSSION TITLE
Other coders's implementations of asynchronous

### DISCUSSION THREAD

**Discussion by Harry-Jing** [acheong08/ChatGPT#210] [Show and tell]
https://github.com/A-kirami/nonebot-plugin-chatgpt/blob/master/nonebot_plugin_chatgpt/chatgpt.py

**Comment by acheong08** [comment]
It looks pretty good. You can reference it and desyncronize it from the standard package as long as the functions and parameters remain the same

**Reply by acheong08** [reply]
One of their pull request shows how to get it working on ARM with session token without crash. Will implement
